src/snowflake/snowpark/modin/plugin/utils/frontend_constants.py

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) 2012-2024 Snowflake Computing Inc. All rights reserved.
+#
+
+# Do not look up certain attributes in columns or index, as they're used for some
```

Comment:
you mentioned we want to take a look to see if we could unify this with the definition in modin, can we get a ticket to track this?

Reply:
SNOW-1643986 (also added to code)